benchmark/README.md

```diff
@@ -292,74 +295,74 @@ Specs: [details](https://www.khadas.com/vim3)
+359.72     336.21     326.91     [224, 224]   PPResNet with ['image_classification_ppresnet50_2022jan_int8.onnx']
+145.21     125.62     124.87     [320, 240]   LPD_YuNet with ['license_plate_detection_lpd_yunet_2023mar.onnx']
+130.10     139.45     116.10     [320, 240]   LPD_YuNet with ['license_plate_detection_lpd_yunet_2023mar_int8.onnx']
+218.21     216.01     199.88     [416, 416]   NanoDet with ['object_detection_nanodet_2022nov.onnx']
+212.69     262.75     170.88     [416, 416]   NanoDet with ['object_detection_nanodet_2022nov_int8.onnx']
+1110.87    1112.27    1085.31    [640, 640]   YoloX with ['object_detection_yolox_2022nov.onnx']
+1128.73    1157.12    1085.31    [640, 640]   YoloX with ['object_detection_yolox_2022nov_int8.onnx']
+382.57     464.42     354.66     [1280, 720]  DaSiamRPN with ['object_tracking_dasiamrpn_kernel_cls1_2021nov.onnx', 'object_tracking_dasiamrpn_kernel_r1_2021nov.onnx', 'object_tracking_dasiamrpn_model_2021nov.onnx']
+147.01     144.01     139.27     [192, 192]   MPPalmDet with ['palm_detection_mediapipe_2023feb.onnx']
+119.70     118.95     94.09      [192, 192]   MPPalmDet with ['palm_detection_mediapipe_2023feb_int8.onnx']
+107.63     107.09     105.61     [224, 224]   MPPersonDet with ['person_detection_mediapipe_2023mar.onnx']
+333.03     346.65     322.37     [128, 256]   YoutuReID with ['person_reid_youtu_2021nov.onnx']
+322.95     315.22     303.07     [128, 256]   YoutuReID with ['person_reid_youtu_2021nov_int8.onnx']
+127.16     173.93     99.77      [256, 256]   MPPose with ['pose_estimation_mediapipe_2023mar.onnx']
+975.49     977.45     952.43     [640, 480]   DB with ['text_detection_DB_IC15_resnet18_2021sep.onnx']
+970.16     970.83     928.66     [640, 480]   DB with ['text_detection_DB_TD500_resnet18_2021sep.onnx']
+194.80     195.37     192.65     [1280, 720]  CRNN with ['text_recognition_CRNN_CH_2021sep.onnx']
+209.49     208.33     192.65     [1280, 720]  CRNN with ['text_recognition_CRNN_CN_2021nov.onnx']
+192.90     227.02     161.94     [1280, 720]  CRNN with ['text_recognition_CRNN_EN_2021sep.onnx']
+192.52     197.03     161.94     [1280, 720]  CRNN with ['text_recognition_CRNN_CH_2023feb_fp16.onnx']
+185.92     168.22     161.94     [1280, 720]  CRNN with ['text_recognition_CRNN_EN_2023feb_fp16.onnx']
+185.01     183.14     161.94     [1280, 720]  CRNN with ['text_recognition_CRNN_CH_2022oct_int8.onnx']
+186.09     194.14     161.94     [1280, 720]  CRNN with ['text_recognition_CRNN_CN_2021nov_int8.onnx']
+181.79     181.65     154.21     [1280, 720]  CRNN with ['text_recognition_CRNN_EN_2022oct_int8.onnx']
 ```
 
 NPU (TIMVX):
 
 ```
-$ python3 benchmark.py --all --int8 --cfg_overwrite_backend_target 3 --model_exclude license_plate_detection_lpd_yunet_2023mar_int8.onnx:human_segmentation_pphumanseg_2023mar_int8.onnx
+$ python3 benchmark.py --all --int8 --cfg_overwrite_backend_target 3 --cfg_exclude face_detection:license_plate
```

Comment:
Model `face_detection_yunet` and `licence_plate_detection_yunet` is excluded because they failed to run:

```
E [/home/opencv-cn/Workspace/opencv/build/release-4.8.0/3rdparty/libtim-vx/TIM-VX-1d9c7ab941b3d8d9c4d28d80058402725731e3d6/src/tim/vx/internal/src/utils/vsi_nn_dtype_util.c:vsi_nn_QuantCheck:429]input_scale[1.000000000000] * weight_scale[0.010799630545] != bias_scale[0.000318158010]
E [/home/opencv-cn/Workspace/opencv/build/release-4.8.0/3rdparty/libtim-vx/TIM-VX-1d9c7ab941b3d8d9c4d28d80058402725731e3d6/src/tim/vx/internal/src/vsi_nn_graph.c:setup_node:481]Check node[4] CONV2D fail
Traceback (most recent call last):
  File "benchmark.py", line 227, in <module>
    benchmark.run(model)
  File "benchmark.py", line 132, in run
    self._benchmark_results[str(size)][filename] = self._metric.forward(model, *data[1:])
  File "/home/opencv-cn/Workspace/opencv/opencv_zoo/benchmark/utils/metrics/detection.py", line 21, in forward
    model.infer(img)
  File "/home/opencv-cn/Workspace/opencv/opencv_zoo/models/face_detection_yunet/yunet.py", line 54, in infer
    faces = self._model.detect(image)
cv2.error: OpenCV(4.8.0) /home/opencv-cn/Workspace/opencv/opencv/modules/dnn/src/op_timvx.cpp:513: error: (-5:Bad argument)  Fail to compile TimVX graph! in function 'forward'


E [/home/opencv-cn/Workspace/opencv/build/release-4.8.0/3rdparty/libtim-vx/TIM-VX-1d9c7ab941b3d8d9c4d28d80058402725731e3d6/src/tim/vx/internal/src/utils/vsi_nn_dtype_util.c:vsi_nn_QuantCheck:429]input_scale[1.000000000000] * weight_scale[0.003745428752] != bias_scale[0.000043810331]
E [/home/opencv-cn/Workspace/opencv/build/release-4.8.0/3rdparty/libtim-vx/TIM-VX-1d9c7ab941b3d8d9c4d28d80058402725731e3d6/src/tim/vx/internal/src/vsi_nn_graph.c:setup_node:481]Check node[4] CONV2D fail
Traceback (most recent call last):
  File "benchmark.py", line 227, in <module>
    benchmark.run(model)
  File "benchmark.py", line 132, in run
    self._benchmark_results[str(size)][filename] = self._metric.forward(model, *data[1:])
  File "/home/opencv-cn/Workspace/opencv/opencv_zoo/benchmark/utils/metrics/detection.py", line 21, in forward
    model.infer(img)
  File "/home/opencv-cn/Workspace/opencv/opencv_zoo/models/license_plate_detection_yunet/lpd_yunet.py", line 54, in infer
    outputBlob = self.model.forward(self.output_names)
cv2.error: OpenCV(4.8.0) /home/opencv-cn/Workspace/opencv/opencv/modules/dnn/src/op_timvx.cpp:513: error: (-5:Bad argument)  Fail to compile TimVX graph! in function 'forward'
```

@zihaomu could you take a look at the problem if possible?